src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionService.ts

```diff
@@ -395,6 +395,21 @@ export class TerminalCompletionService extends Disposable implements ITerminalCo
 				}
 			}
 
+			// Try to resolve symlink target for symbolic links
+			let symlinkIsDirectory: boolean;
+			if (child.isSymbolicLink) {
+				try {
+					symlinkIsDirectory = await this._symlinkIsDirectory(child.resource);
+					if (symlinkIsDirectory) {
+						kind = TerminalCompletionItemKind.Folder;
+					} else {
+						kind = TerminalCompletionItemKind.File;
+					}
```

Comment:
Would it be possible to include the already established symlink icon in addition to the file/folder icon?

![image](https://github.com/user-attachments/assets/78d2bab7-c815-4588-aa7b-aa1a32270f3b)

We actually have just file+folder symlink icons, which would maybe be better than the above since I don't think we can nicely have 2 icons:

![image](https://github.com/user-attachments/assets/15227078-6ee2-4825-ad21-3519ac645469)